### ISSUE TITLE
Snapping: Moving blocks in relation to the page

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amp-wp#2993]
See #2875 for details.

Example: if moving around the center, snapping should enable positioning exactly to the center of the page.

**Comment by swissspidy** [comment]
See #2992 for screencast.

**Comment by swissspidy** [comment]
**Testing Instructions**

1. Create new story
1. Move the pre-existing block around on the page
1. For each of the block's sides (left, right, top, bottom) and centers (vertical and horizontal), verify that red snapping guidelines are shown in the following scenarios:  
    1. When moving the block to the sides of the page (left, right, top, bottom).
    1. When moving the block to the center (vertical and horizontal) of the page.

Basically, snapping guidelines should be shown to allow easily aligning blocks to the sides and center of the page. No actual _snapping_ is done.

Next, verify that no guidelines are shown when dragging and at the same time holding down the alt/option key.

**Comment by csossi** [comment]
Verified in QA
